src/configuration/prima.rs

```diff
@@ -29,6 +30,10 @@ impl PrimaConfiguration {
         self.tags.push(format!("{}:{}", key, value));
         self
     }
+
+    pub fn country(self, country: Country) -> Self {
```

Comment:
I wonder if there's a better name because `country` makes me think I'm going to get a country

Maybe `with_country`? `with_country_tag`? :thinking:

Reply:
Changed to `with_country`